src/promptflow/promptflow/contracts/tool.py

```diff
@@ -252,6 +252,7 @@ class InputDefinition:
     # For a custom strong type connection input, the type should be 'CustomConnection',
     # while the custom_type should be the custom strong type connection class name.
     custom_type: List[str] = None
+    ui_hints: dict = None
```

Comment:
please add comment for this new parameter.

Reply:
Added, thanks.